sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -324,17 +333,20 @@ class SQLMetricsSuite extends SharedSparkSession with SQLMetricsTestUtils {
       // +- Exchange(nodeId = 5)
       // +- Project(nodeId = 6)
       // +- LocalTableScan(nodeId = 7)
-      val df = df1.join(df2, "key")
-      testSparkPlanMetrics(df, 1, Map(
-        1L -> (("ShuffledHashJoin", Map(
-          "number of output rows" -> 2L))),
-        2L -> (("Exchange", Map(
-          "shuffle records written" -> 2L,
-          "records read" -> 2L))),
-        5L -> (("Exchange", Map(
-          "shuffle records written" -> 10L,
-          "records read" -> 10L))))
-      )
+      Seq((1L, 2L, 5L, false), (2L, 3L, 7L, true)).foreach{args =>
```

Comment:
nit: `.foreach{args =>` -> `.foreach { args =>`

Reply:
fixed in new the new commit